src/app/components/Footer/Footer.scss

```diff
@@ -0,0 +1,177 @@
+                    display: flex;
+                    align-items: center;
+                    gap: 10px;
+
+                    img {
+                        width: 177px;
+                        height: 45px;
+                    }
+
+                    p {
+                        font-family: $header-font;
+                        font-style: normal;
+                        font-weight: 400;
+                        font-size: 40px;
+                        line-height: 125%;
+                        display: flex;
+                        align-items: center;
+                        margin-block-start: 0;
+                        margin-block-end: 0;
+                    }
+                }
+
+                .text {
+                    font-family: $common-writing-font;
+                    font-style: normal;
+                    font-weight: 400;
+                    font-size: 22px;
+                    line-height: 150%;
+                }
+            }
+
+            .icons {
+                width: 52%;
+                display: flex;
+                justify-content: space-around;
+
+                button {
+                    border-style: none;
+                    background-color: #FFFF;
+                }
+            }
+        }
+
+
+        .main-pages {
+            display: flex;
+            flex-direction: column;
+            width: 10%;
+
+            p {
+                font-family: 'DM Serif Display';
+                font-style: normal;
+                font-weight: 400;
+                font-size: 25px;
+                line-height: 125%;
+                color: rgba(41, 47, 54, 1);
+
+            }
+
+            .pages {
+                display: flex;
+                flex-direction: column;
+                gap: 9px;
+
+                .link {
+                    text-decoration: none;
+                    font-family: 'Jost';
+                    font-style: normal;
+                    font-weight: 400;
+                    font-size: 22px;
+                    line-height: 300%;
+                    color: #4D5053;
+                }
+            }
+        }
+
+        .main-service {
+            display: flex;
+            flex-direction: column;
+
+            p {
+                font-family: 'DM Serif Display';
+                font-style: normal;
+                font-weight: 400;
+                font-size: 25px;
+                line-height: 125%;
+                color: rgba(41, 47, 54, 1);
+
+            }
+
+            .service {
+                display: flex;
+                flex-direction: column;
+                gap: 9px;
+
+                .link {
+                    text-decoration: none;
+                    font-family: 'Jost';
+                    font-style: normal;
+                    font-weight: 400;
+                    font-size: 22px;
+                    line-height: 300%;
+                    color: #4D5053;
+                }
+            }
+        }
```

Comment:
you are repeating yourself
please check styles at lines 85-110
they are same
so you need to create only one css class